static/header.less

```diff
@@ -33,7 +33,7 @@
             width: 100%;
             cursor: pointer;
             background-image: url(img/canjs_logo.svg);
-            background-size: cover;
+            background-size: 30px;
```

Comment:
Could you please include before & after screenshots to show what this change does?